### ISSUE TITLE
Debian 8 Issue

### ISSUE THREAD

**Issue by rchit3ct** [maddinat0r/samp-discord-connector#17]
Error log:
> [12:14:07] Loading plugin: discord-connector.so [12:14:07] Failed (libssl.so.10: cannot open shared object file: No such file or directory) [12:14:07] Loaded 8 plugins.

I have Debian 8 and the library installed. It throws that error when tried to run on that environment. However, it works fine when run in a Windows environment (with the .dll) on a local setup.

Any fix to this?

**Update:**
It seems the particular issue is that I can't get to install the x32 openssl library, something which I found to be an issue that a user sorted out that he fixed simply by switching to Ubuntu (https://github.com/maddinat0r/samp-discord-connector/issues/16).

To put it simply, after attempting to install openssl, I get a prompt that says that it can be installed, but only at the expense of other packages, which are vital. Here's a screenshot of it: http://puu.sh/w05OU/4e58cdd91b.jpg

I'm looking more into this issue and will definitely post here if I find a solution that does not involve making the switch from one platform to the other.

**Update 2:**

Fixed it, issue can be closed.

**Comment by CXdur** [comment]
For other people with this issue I suggest trying the following package: openssl-dev:i386

**Comment by viruxe** [comment]
What was the fix?

**Comment by rchit3ct** [comment]
@viruxe Look above, a different package was used. 

Still got some issues (#18)
